index.js

```diff
@@ -166,13 +166,13 @@ module.exports.processV2 = (output, origin) => {
   throw 'no url';
 };
 
-module.exports.processV3 = (output, origin) => {
+module.exports.processV3 = (output, origin, locales = []) => {
```

Comment:
would it be better to use the en list as a default here?

Reply:
I guess you can but I've used the merge approach which is too append the EN_LIST to locales. Doesn't make the difference here but could be helpful when we want to give the user the options.